### ISSUE TITLE
Incorrect Results When Running Pre-trained Model Locally

### ISSUE THREAD

**Issue by C-hongfei** [lpiccinelli-eth/UniDepth#81]
Hi, thank you for your excellent work on this project! 
When I downloaded the pre-trained model to my local environment and ran the demo, I encountered incorrect results. The ARel metric is showing 72.07%, which seems to be much higher than expected. Below are the changes I made. Could you please help me identify what might be causing this issue?
Changes Made:
hubconf.py,line33,
`# path = huggingface_hub.hf_hub_download(repo_id=f"lpiccinelli/unidepth-{version}-{backbone}", filename=f"pytorch_model.bin", repo_type="model")
        path = f"lpiccinelli/unidepth-{version}-{backbone}"+"/pytorch_model.bin"`
unidepth/models/backbones/dinov2.py, line451,
`    elif pretrained is not None:
        state_dict = torch.load('dinov2_vits14_pretrain.pth', map_location="cpu")
        info = model.load_state_dict(state_dict, strict=False)`

![output](https://github.com/user-attachments/assets/72c9adfd-6496-4814-b384-4c84547aca2b)

**Comment by furui123** [comment]
> 您好，感谢您在这个项目上出色的工作！ 当我将预训练模型下载到本地环境并运行演示时，出现了错误的结果。ARel 指标显示 72.07%，似乎比预期高得多。以下是我所做的更改。您能帮我找出可能导致此问题的原因吗？ 更改内容： hubconf.py，第 33 行 ![image](https://github.com/user-attachments/assets/5b4148f7-a64e-4c59-93a3-285757c2ea1a)
> 
> unidepth/models/backbones/dinov2.py，第 451 行， 结果： ![image](https://github.com/user-attachments/assets/bb981e98-84cc-4c9d-9780-6abfe0a4b988) ![output](https://github.com/user-attachments/assets/72c9adfd-6496-4814-b384-4c84547aca2b)

Hi, have you solved this problem?
